document/core/exec/modules.rst

```diff
@@ -771,7 +771,7 @@ The values :math:`\val_{\F{res}}^m` are returned as the results of the invocatio
    ~\\[-1ex]
    \begin{array}{@{}lcl}
    \invoke(S, \funcaddr, \val^n) &=& S; F; \val^n~(\INVOKE~\funcaddr) \\
-     &(\iff & S.\SFUNCS[\funcaddr].\FITYPE = [t_1^n] \to [t_2^m] \\
-     &\wedge& \val^n = (t_1.\CONST~c)^n) \\
+     &\wedge& & S.\SFUNCS[\funcaddr].\FITYPE = [t_1^n] \to [t_2^m] \\
+     &\wedge& \val^n = (t_1.\CONST~c)^n \\
```

Comment:
Instead of removing the paren altogether, it should go one line down.

Reply:
Yes, good catch.